src/main/charts/confluence/values.yaml

```diff
@@ -183,6 +183,18 @@ synchrony:
     periodSeconds: 1
     # -- The number of consecutive failures of the Synchrony container readiness probe before the pod fails readiness checks
     failureThreshold: 30
+  resources:
+    jvm:
+      # -- The maximum amount of heap memory that will be used by the Synchrony JVM
+      maxHeap: "1g"
+      # -- The minimum amount of heap memory that will be used by the Synchrony JVM
+      minHeap: "1g"
+      # -- The memory allocated for the Synchrony stack
+      stackSize: "2048k"
+    container: 
```

Comment:
Are these new values documented anywhere?

Reply:
Not really; `-Xmx` is pretty standard, and the stack size (`-Xss`) is in template `start-synchrony.sh` script that you're required to edit by hand. I've just ported them across.

Reply:
Hm yes but these are technically new values you introduced, and they should be documented in `src/main/charts/confluence/README.md` right?

Reply:
The `README.md` is automatically generated from the `values.yaml` file with https://github.com/atlassian-labs/data-center-helm-charts/blob/master/src/main/scripts/generate_helm_docs.sh
So the main thing is to document everything in the `values.yaml` files and we generate the docs later. Right now it is before release but should be automated.

Reply:
Does it make sense to use the same values as connie? They are based on the confluence container jvm specs: https://bitbucket.org/atlassian-docker/docker-atlassian-confluence-server/src/master/

Reply:
I think we should have separate JVM arg values for Synchrony and Connie. I've seen that large customers spend a lot of time fine-tuning their instances and they will be upset if we don't support separate configuration for Synchrony and Confluence JVM. The typical example for enterprise customer would be to give Connie 16Gb of heap and Synchrony 4Gb of heap - they can be different by the order of magnitude.